ArtCeil.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="Styles/Style.css">
+  <title>Потолочные покрытия</title>
+</head>
+
+<body>
+  <header class="header">
+    <div class="header__top">
+      <div class="header__top top__menu">
+        <div class="header__orangeline"></div>
+        <div class="header__greenline"></div>
+        <div class="header__blueline"></div>
+        <div class="header__menu">МЕНЮ</div>
+      </div>
+      <div class="header__contacts">
+        <div>Email:<span class="header__contacts_content">shiningfinger@list.ru</span></div>
+        <div>Телефон:<span class="header__contacts_content">+7 (985) 905-12-51</span></div>
+      </div>
+      <div class="header__logo">
+        <img class="header__logo-image" src="../Artceil/Images/logo.png">
```

Comment:
Урлы стоит указывать относительно той директории, где находится хтмл файл. Тут нет необходимости переходить в предыдущую директорию, заходить обратно и искать картинку

```suggestion
        <img class="header__logo-image" src="./Images/logo.png">
```

Поменяй везде так